lib/ntop.c

```diff
@@ -44,10 +44,13 @@ static inline void putbyte(uint8_t bytex, char **posx)
 		*pos++ = '2';
 		zero = true;
 		byte = tmp;
-	} else if ((tmp = byte - 100) >= 0) {
-		*pos++ = '1';
-		zero = true;
-		byte = tmp;
+	} else {
```

Comment:
hmm - want to do the one at the start of this clause also?

Reply:
missed that, thanks :+1: